{
  "id": "CVE-2017-9800",
  "definition_id": "oval:org.opensuse.security:def:20179800",
  "title": "CVE-2017-9800",
  "description": "\n    A maliciously constructed svn+ssh:// URL would cause Subversion clients before 1.8.19, 1.9.x before 1.9.7, and 1.10.0.x through 1.10.0-alpha3 to run an arbitrary shell command. Such a URL could be generated by a malicious server, by a malicious user committing to a honest server (to attack another user of that server's repositories), or by a proxy server. The vulnerability affects all clients, including those that use file://, http://, and plain (untunneled) svn://.\n    ",
  "cvss3": "8.1/CVSS:3.0/AV:N/AC:H/PR:N/UI:N/S:U/C:H/I:H/A:H",
  "severity": "Important",
  "impact": "important",
  "affected": {
    "family": "unix",
    "platforms": [
      "openSUSE Tumbleweed"
    ],
    "cpes": [
      "cpe:/o:opensuse:tumbleweed"
    ]
  },
  "packages": [
    {
      "name": "libsvn_auth_gnome_keyring-1-0",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "libsvn_auth_kwallet-1-0",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion-bash-completion",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion-devel",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion-perl",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion-python",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion-ruby",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion-server",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    },
    {
      "name": "subversion-tools",
      "status": "released",
      "fixed_version": "0:1.14.1-1.11",
      "platform": "openSUSE Tumbleweed"
    }
  ],
  "references": [
    {
      "id": "Mitre CVE-2017-9800",
      "source": "CVE",
      "url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-9800"
    },
    {
      "id": "SUSE CVE-2017-9800",
      "source": "SUSE CVE",
      "url": "https://www.suse.com/security/cve/CVE-2017-9800"
    },
    {
      "id": "SUSE-SU-2017:2163-1",
      "source": "SUSE-SU",
      "url": "https://lists.opensuse.org/opensuse-security-announce/2017-08/msg00045.html"
    },
    {
      "id": "SUSE-SU-2017:2200-1",
      "source": "SUSE-SU",
      "url": "https://lists.opensuse.org/opensuse-security-announce/2017-08/msg00055.html"
    },
    {
      "id": "openSUSE-SU-2017:2183-1",
      "source": "SUSE-SU",
      "url": "https://lists.opensuse.org/opensuse-security-announce/2017-08/msg00052.html"
    },
    {
      "id": "SUSE bug 1051362",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1051362"
    },
    {
      "id": "SUSE bug 1052481",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1052481"
    },
    {
      "id": "SUSE bug 1052696",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1052696"
    },
    {
      "id": "SUSE bug 1052932",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1052932"
    },
    {
      "id": "SUSE bug 1053364",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1053364"
    },
    {
      "id": "SUSE bug 1054653",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1054653"
    },
    {
      "id": "SUSE bug 1066430",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1066430"
    },
    {
      "id": "SUSE bug 1071709",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1071709"
    },
    {
      "id": "SUSE bug 1128150",
      "source": "BUG",
      "url": "https://bugzilla.suse.com/1128150"
    }
  ],
  "issued": "2021-10-02T00:00:00Z",
  "updated": "2021-10-02T00:00:00Z"
}
